packages/wicked/1007-include-mac-address-in-leaseinfo.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/leaseinfo.c b/src/leaseinfo.c
+index c47241f5..8a827ef7 100644
+--- a/src/leaseinfo.c
++++ b/src/leaseinfo.c
+@@ -692,6 +692,23 @@ __ni_leaseinfo_dump(FILE *out, const ni_addrconf_lease_t *lease,
+ {
+ 	unsigned int i;
+ 	char *key = NULL;
++	ni_netconfig_t *nc = ni_global_state_handle(0);
```

Comment:
This assignment is clobbered by the second call, so you should remove it:
```suggestion
+	ni_netconfig_t *nc;
```

Reply:
Probably borrowed from other code that calls `ni_global_state_handle()`, there seems to be similar clobbering elsewhere in wicked.